src/unstable-temp/DownloadImageModal/ImageForm.tsx

```diff
@@ -46,6 +47,16 @@ const getDeviceTypeOptions = (deviceType: DeviceType) => {
 			});
 		}
 
+		// Add extra input field to advanced config
+		if (group.name === 'advanced') {
+			group.options.push({
+				message: t('info.provisioning_key_name'),
+				name: 'provisioningKeyName',
+				default: '',
```

Comment:
NIT: Could this be `null` in case the user didn't care to provide any?

Reply:
You mean the `group` can be `null` here? We are already putting an empty string as default.